qt5-layer/recipes-qt/qt5/qtwayland_git.bbappend

```diff
@@ -0,0 +1,2 @@
+# when tracing is enabled, qtwayland sources need to link with libdl
+OE_QMAKE_LINK += " -ldl "
```

Comment:
Why the extra spaces? " -ldl " -> "-ldl"

Reply:
yup, that's unnecessary